### DISCUSSION TITLE
Change Terminal Color Scheme by connected host

### DISCUSSION THREAD

**Discussion by amie42** [microsoft/terminal#14721] [General]
Hi, 

did anyone already manage to change the Terminal Appearance depending on the connected host? I am looking for a more annoying Scheme while connected to PROD hosts so I won't hang around on theses machines for any longer than required. And also to stay alerted to not mess around with these shells. 

Any help would be appreciated 

Amie

**Comment by zadjii-msft** [comment]
I mean, you could always just make different profiles for each host, and change the colorscheme of those profiles. 

I dunno if there's a good way to have the scheme change automatically in response to you ssh'ing from another shell though. I can't imagine there's a good way for the Terminal to be able to detect that. The best chance might be to do some action in response to the text `ssh me@my-prod-server.whatever`, but that wouldn't be able to switch back after dropping out of ssh.

**Reply by amie42** [reply]
Thanks for that super-fast answer. I've seen that feature at colleagues' screens on Mac and Linux Boxes. Might be they implemented it using tmux or similar environments. But I think it might be a cool feature for your product as well. 

I would btw parse the prompt to distinguish between environments.

Best
Amie
